packages/commonwealth/client/scripts/views/components/component_kit/new_designs/search/dropdown.tsx

```diff
@@ -0,0 +1,241 @@
+import React, { FC } from 'react';
+import moment from 'moment';
+
+import NewProfilesController from '../../../../../controllers/server/newProfiles';
+import AddressInfo from '../../../../../models/AddressInfo';
+import ChainInfo from '../../../../../models/ChainInfo';
+import { SearchScope } from '../../../../../models/SearchQuery';
+import { useCommonNavigate } from 'navigation/helpers';
+import { CommunityLabel } from '../../../community_label';
+import { CWText } from '../../cw_text';
+import { User } from '../../../user/user';
+import { QuillRenderer } from '../../../react_quill_editor/quill_renderer';
+import { renderTruncatedHighlights } from '../../../react_quill_editor/highlighter';
+import { CWDivider } from '../../cw_divider';
+import {
+  CommunityResult,
+  MemberResult,
+  ReplyResult,
+  ThreadResult,
+} from '../../../../pages/search/helpers';
+
+import '../../../../../../styles/components/component_kit/new_designs/CWSearchBar.scss';
+
+type SearchBarThreadPreviewRowProps = {
+  searchResult: ThreadResult;
+  searchTerm?: string;
+};
+
+type SearchBarCommentPreviewRowProps = {
+  searchResult: ReplyResult;
+  searchTerm?: string;
+};
+
+type SearchBarChainPreviewRowProps = {
+  searchResult: CommunityResult;
+  searchTerm?: string;
+};
+
+type SearchBarMemberPreviewRowProps = {
+  searchResult: MemberResult;
+  searchTerm?: string;
+};
+
+type SearchBarPreviewSectionProps = {
+  searchResults: any;
+  searchTerm: string;
+  searchScope: SearchScope;
+};
+
+type SearchBarDropdownProps = {
+  searchTerm: string;
+  searchResults: any;
+};
+
+const SearchBarThreadPreviewRow = (props: SearchBarThreadPreviewRowProps) => {
```

Comment:
this is not aligned with our new standards described here
https://github.com/hicommonwealth/commonwealth/blob/master/wiki/React-Best-Practices-And-Improvements.md#directory-structure

- each component in own file
- each file has its own css file with the same name, and the top className aligned with file/component name
- component name is the same as file name
- all of components should be imported to and exported from index file

Reply:
Addressed.

Reply:
looks much better!